src/features/Courses/data/thunks.js

```diff
@@ -7,12 +7,12 @@ import {
 } from 'features/Courses/data/slice';
 import { getCoursesByInstitution } from 'features/Common/data/api';
 
-function fetchCoursesData(id, currentPage, filtersData) {
+function fetchCoursesData(id, currentPage, filtersData, licenseId='') {
```

Comment:
Same comment about the params 

```suggestion
function fetchCoursesData(id, currentPage, filtersData) {
```